Example/Podfile.lock

```diff
@@ -6,11 +6,11 @@ DEPENDENCIES:
 
 EXTERNAL SOURCES:
   Sonar:
-    :path: ../
+    :path: "../"
 
 SPEC CHECKSUMS:
-  Sonar: a49fe87e37bb69d965d21f51593ede4f271402fc
+  Sonar: 0f8c975b011d1c5e7ec3a66997957844d7ea8dd5
 
-PODFILE CHECKSUM: 5835228cd562b6041ea376b55a808091c42f184a
+PODFILE CHECKSUM: b85457d725335c87745002db0612cea5101858a0
 
-COCOAPODS: 1.2.0
+COCOAPODS: 1.6.0.beta.1
```

Comment:
The lockfile is generated by different version, in Gemfile 1.5.3 is required.

Reply:
Hello @mkj-is 
Usually with projects created for Xcode 10 I use CocoaPods 1.6.0.beta.1. However this is a beta version of CocoaPods, so I decided to add CocoaPods 1.5.3 to the Gemfile. What version of CocoaPods are you using now?

Reply:
I agree 1.6.0 works better with Xcode 10. This is really minor issue, we do not need to fix this. Or I can make other PR cleaning this up, this might be easier.

Reply:
Hello @mkj-is 
I added CocoaPods 1.6.0.beta.1 as a dependency.